npep/npep-122.md

```diff
@@ -0,0 +1,194 @@
+# NPEP-122: NPEP template
+
+* Issue: [#122](https://github.com/kubernetes-sigs/network-policy-api/issues/122)
+* Status: Provisional
+
+## TLDR
+
+Stop using SameLabels/NotSameLabels to define tenancy within ANP. Instead, propose a separate object
+that will define Tenancy and ensure that subject and peer relation is explicitly defined.
```

Comment:
I agree too :) there may be some artifacts like that from the first version of npep that was focused on the specific API changes instead of user stories, let me clean this up